src/ert/_c_wrappers/job_queue/workflow_job.py

```diff
@@ -46,8 +43,29 @@ class WorkflowJob(BaseCClass):
     )
 
     @classmethod
-    def configParser(cls):
-        return cls._alloc_parser()
+    def configParser(cls) -> ConfigParser:
+        parser = ConfigParser()
+        parser.add("MIN_ARG", value_type=ContentTypeEnum.CONFIG_INT).set_argc_minmax(
+            1, 1
+        )
+        parser.add("MAX_ARG", value_type=ContentTypeEnum.CONFIG_INT).set_argc_minmax(
+            1, 1
+        )
+        parser.add(
+            "EXECUTABLE", value_type=ContentTypeEnum.CONFIG_EXECUTABLE
+        ).set_argc_minmax(1, 1)
+        parser.add("SCRIPT", value_type=ContentTypeEnum.CONFIG_PATH).set_argc_minmax(
+            1, 1
+        )
+        parser.add("FUNCTION").set_argc_minmax(1, 1)
+        parser.add("INTERNAL", value_type=ContentTypeEnum.CONFIG_BOOL).set_argc_minmax(
+            1, 1
+        )
+        item = parser.add("ARG_TYPE")
+        item.set_argc_minmax(2, 2)
+        item.iset_type(0, ContentTypeEnum.CONFIG_INT)
+        item.initSelection(1, ["STRING", "INT", "FLOAT", "BOOL"])
```

Comment:
make this purer? by chaining the calls, instead of repeating `item`?

Reply:
Would mean changing all the setters so a bit outside scope, but I think using builder pattern for SchemaItem makes sense.